AppInspector/Commands/VerifyRulesCommand.cs

```diff
@@ -136,20 +136,9 @@ public VerifyRulesResult GetResult()
             try
             {
                 RulesVerifier verifier = new RulesVerifier(_rules_path, _options.Log);
-                verifier.Verify(_options.Failfast);
                 verifyRulesResult.ResultCode = VerifyRulesResult.ExitCode.Verified;
```

Comment:
Do you want rules to be considered verified if the verification throws an exception?

Reply:
There's already an exception handler and the VerifyRulesResult is local so setting it's value for a critical event doesn't help.  Verified and NotVerified (non-critical i.e. file loads but Rule is not compliant) does return an error to either CLI or Nuget callers.  Both the CLI and Nuget callers are expected to try/catch critical errors.

Reply:
So why set the result code at all?  You discard this value in all cases.  Just leave it the default enum value until you properly set it.